src/Api/PubnubApi/Security/PubnubCryptoBase.cs

```diff
@@ -163,24 +156,11 @@ public string PubnubAccessManagerSign(string key, string data)
             byte[] keyByte = encoding.GetBytes(secret);
             byte[] messageBytes = encoding.GetBytes(message);
 
-#if NET35
             using (var hmacsha256 = new HMACSHA256(keyByte))
             {
                 byte[] hashmessage = hmacsha256.ComputeHash(messageBytes);
                 return Convert.ToBase64String(hashmessage).Replace('+', '-').Replace('/', '_');
             }
```

Comment:
Can you please confirm if this code works with UWP?

Reply:
Yes. It works on UWP as well. 
Tested and here is the log of request and response with signature
DateTime 11/18/2021 12:00:39, JSON= {"message":"Success","payload":{"level":"channel","subscribe_key":"sub-c-c9710928-1b7a-11e3-a0c8-02ee2ddab7fe","ttl":1440,"channels":{"my_channel_pandu":{"r":1,"w":1,"m":0,"d":0,"g":0,"u":0,"j":0}}},"service":"Access Manager","status":200} for request=http://ps.pndsn.com/v2/auth/grant/sub-key/sub-c-c9710928-1b7a-11e3-a0c8-02ee2ddab7fe?channel=my_channel_pandu&d=0&g=0&j=0&m=0&pnsdk=NETFX_CORECSharp5.4.0.0&r=1&requestid=1e2c1a26-85a0-442f-81f9-719773773a22&timestamp=1637217039&ttl=1440&u=0&uuid=winphoneuuid&w=1&signature=v2.exZfJvN_40a3zop-NNJ1CugHLgpXRHoKTAPYTfO-XYE